src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/StackExchangeRedis/Instrumentation.xml

```diff
@@ -6,11 +6,11 @@ SPDX-License-Identifier: Apache-2.0
 <extension xmlns="urn:newrelic-extension">
   <instrumentation>
     <tracerFactory>
-      <match assemblyName="StackExchange.Redis" className="StackExchange.Redis.ConnectionMultiplexer">
+      <match assemblyName="StackExchange.Redis" className="StackExchange.Redis.ConnectionMultiplexer" minVersion="1.0.0.0" maxVersion="2.0.0.0">
```

Comment:
It doesn't hurt anything to have it, but you could leave off `minVersion` here if you want.

Reply:
Yeah, I waffled back and forth on that, but in the end I had only tested with 1.x and above so I went with being more specific.